### DISCUSSION TITLE
How to send specific jobs to specific workers?

### DISCUSSION THREAD

**Discussion by assembly-winston** [riverqueue/river#683] [Q&A]
Hi everyone! Trying to figure out how to enable a "dev-only" worker for testing/staging changes. Would this need to be run on a completely separate postgres db? Or is there a way to "version" or otherwise specify which workers to send specific jobs to?

**Comment by travisby** [comment]
If you have "dev only" jobs, you could just not register the worker in your production process https://riverqueue.com/docs#registering-workers

If you want to use the same job args, then you could use a separate queue https://riverqueue.com/docs/multiple-queues and only have the dev worker listen to that queue
